geomstats/geometry/matrices.py

```diff
@@ -143,7 +161,19 @@ def __init__(self, m, n):
             signature=(dimension, 0, 0))
 
     def inner_product(self, tangent_vec_a, tangent_vec_b, base_point=None):
-        """Compute Frobenius inner product of two tan vecs at `base_point`."""
+        """Compute Frobenius inner product of two tan vecs at `base_point`.
+
+        Parameters
+        ----------
+        tangent_vec_a : array-like, shape=[n_samples, m, n]
+        tangent_vec_b : array-like, shape=[n_samples, m, n]
+        base_point : array-like, shape=[n_samples, m, n], optional
```

Comment:
Descriptions? You can copy paste from PR #425 , I have docstrings for inner products there.